precompiles/randomness/Randomness.sol

```diff
@@ -64,9 +64,9 @@ interface Randomness {
         uint32 numWords;
         RandomnessSource randomnessSource;
         uint32 fulfillmentBlock;
-        uint32 fulfillmentEpochIndex;
+        uint64 fulfillmentEpochIndex;
```

Comment:
isnt this changing the return type of getRequest? I guess we are good with it because it is potentially not being widely used at the moment but wouldnt this break compatibility?

Reply:
Sort of. The return type isn't taken into account when computing the selector. However contracts already deployed that expects a `uint32` will just don't care and truncate the value. When writing manually the precompile impl was already writing u64 instead of u32, and we didn't saw it because there was no type checking for that.